backend/src/email/services/custom-domain.service.ts

```diff
@@ -143,9 +153,18 @@ const storeFromAddress = async (
  * 3. Checks the domain's dns to ensure that the cnames are there
  */
 const verifyFromAddress = async (email: string): Promise<void> => {
-  const dkimTokens = await verifyEmailWithAWS(email)
+  const dkimTokensBackend = await verifyEmailWithAWS(
+    email,
+    backendSes,
+    backendSesRegion
+  )
+  const dkimTokensWorker = await verifyEmailWithAWS(
+    email,
+    workerSes,
+    workerSesRegion
+  )
```

Comment:
Rewrite this to 
```ts
await Promise.all([
  verifyEmailWithBackend,
  verifyEmailWithWorker,
])
```
to make more efficient use of the event loop cycle

Reply:
Ohh good catch. Done.